firmware/shared/controls/tvFactor.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
+#include <cmath>
+#include <vector>
+#include <algorithm>
+#include <tuple>
+#include "../util/mappers/lookup_table.h"
+
+namespace ctrl{
+
+template <typename T>
+T tvEnable(T DI_p_steeringAngle, T DI_b_tvEnable){
+
+    if (DI_b_tvEnable > 0) return DI_p_steeringAngle;
+    else return 0;
+}
+
+template <typename T>
+T open_loop_tv_lookUp(T DI_p_steeringAngle){
+
+    T Angle = std::abs(DI_p_steeringAngle);
+
+    // Lookup Table Data
+    // breakpoints = {0, 5, 10, 15, 20, 25}; 
+    // tableData = {1, 0.934, 0.87, 0.808, 0.747, 0.683};
+
+    const float steeringAngle_lut_data[][2] = {
+    {0.0, 1.0f},
+	{5.0, 0.934f},
+	{10.0, 0.87f},
+	{15.0, 0.808f},
+	{20.0, 0.747f},
+	{25.0, 0.683f},
+    };
+
+    constexpr int steeringAngle_lut_length =
+    (sizeof(steeringAngle_lut_data)) / (sizeof(steeringAngle_lut_data[0]));
+
+    shared::util::LookupTable<steeringAngle_lut_length> steeringAngle_adc_lut{steeringAngle_lut_data};
+
+    T result = steeringAngle_adc_lut.Evaluate(Angle);
```

Comment:
this `T result = ...` and `return result` can be inlined, no need to create a variable